cli/commands/start.go

```diff
@@ -57,7 +57,7 @@ func init() {
 func runStartCmd(cmd *cobra.Command, args []string) error {
 	var err error
 
-	if timeoutStr == "" && !ctx.Running.Daemonize {
+	if timeoutStr != "" && !ctx.Running.Daemonize {
```

Comment:
It's a behaviour change. But seems it's not tested anyhow and not described in changelog...
Also looks like it's not related to your current patch and should be pushed separately.

Reply:
https://github.com/tarantool/cartridge-cli/pull/471